docs/tutorials/rigetti/getting_started.ipynb

```diff
@@ -556,6 +624,7 @@
  ],
  "metadata": {
   "colab": {
+   "collapsed_sections": [],
```

Comment:
not sure how this relates, but maybe this line is what triggered the metadata error.

Reply:
No 🎲 .

I've tried to set the following metadata stanza based on `example.ipynb` but running `check/nbformat --apply` blows the changes away. I've also rebased from master and recreated an brand new virtual env.

```json
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
```